src/tray/templates/services/app/deploy.yml.j2

```diff
@@ -67,14 +67,18 @@ spec:
             initialDelaySeconds: 10
             periodSeconds: 5
           env:
+            - name: CORS_ALLOWED_ORIGINS
+              value: "{{ richie_host | blue_green_hosts | split(',') | map('regex_replace', '^(.*)$', 'https://\\1') | join(',') }}"
             - name: DB_HOST
               value: "joanie-{{ joanie_database_host }}-{{ deployment_stamp }}"
             - name: DB_NAME
               value: "{{ joanie_database_name }}"
             - name: DB_PORT
               value: "{{ joanie_database_port }}"
             - name: DJANGO_ALLOWED_HOSTS
-              value: "{{ joanie_host }}"
+              value: "{{ joanie_host | blue_green_hosts }}"
+            - name: DJANGO_CSRF_TRUSTED_ORIGINS
+              value: "{{ joanie_host | blue_green_hosts | split(',') | map('regex_replace', '^(.*)$', 'https://\\1') | join(',') }}"
```

Comment:
And richie hosts should not be added here ?

Reply:
Currently this is not needed but surely in the futur.